src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7195,9 +7087,9 @@ bool canFollowNullableType()
                         // e.g. `new int?()` is allowed.  It creates a null value of type `Nullable<int>`.
                         // Similarly `new int? {}` is allowed.
                         return
-                            this.CurrentToken.Kind == SyntaxKind.OpenParenToken ||   // ctor parameters
-                            this.CurrentToken.Kind == SyntaxKind.OpenBracketToken ||   // array type
-                            this.CurrentToken.Kind == SyntaxKind.OpenBraceToken;   // object initializer
+                            this.CurrentToken.Kind is SyntaxKind.OpenParenToken or   // ctor parameters
+                                                      SyntaxKind.OpenBracketToken or   // array type
```

Comment:
I personally would not do this (inserting extra spaces in order to make certain tokens align across lines).

In fact, this is something the format analyzer forbids in other cases. e.g. https://github.com/dotnet/roslyn/blob/d73d31cbccb9aa850f3582afb464b709fef88fd7/src/Workspaces/Core/Portable/Editing/DeclarationModifiers.cs#L192-L216

I would much rather see us simply insert a line break + indent before the token where we desire the alignment.

```cs
return this.CurrentToken.Kind is
    SyntaxKind.OpenParenToken or
    SyntaxKind.OpenBracketToken or
    SyntaxKind.OpenBraceToken;
```

I prefer even more to put the operator token at the start of the line. SyntaxKind will not strictly align in this case (if we used an `and` or `not` operator in some places, for example), but the visual pop of seeing the smaller operator token at the start of the line seems better for skimmability, in my opinion.

```cs
return this.CurrentToken.Kind
    is SyntaxKind.OpenParenToken
    or SyntaxKind.OpenBracketToken
    or SyntaxKind.OpenBraceToken;
```

Reply:
i can do that.  i checked with IDE at one point if we had a view on where was best for operators, and currently there's no consistency from the team.  so we'ved decided to not be opinionated there.